server/db/index.js

```diff
@@ -258,6 +258,7 @@ const seed = async (force = true) => {
       await sync();
     }
     console.log('seed was successful');
+    process.exit();
```

Comment:
I don't think this is a good idea, according to the docs this could stop the process before all of our test data is created.

https://nodejs.org/api/process.html#process_process_exit_code

Reply:
should be ok since we're awaiting all the data before it first, elliot said in the lecture we need a way to exit the sync, atm we do need to control c it i believe